backend/src/test/java/de/neuefische/service/TankServiceTest.java

```diff
@@ -61,4 +60,14 @@ void addTank_shouldAddNewTank() {
         //then
         assertEquals(actual, tank);
     }
+
+    @Test
+    void deleteTank_shouldRemoveTank() {
+        //given
+        String testId = "1";
+        //when
+        tankService.deleteTank(testId);
+        //then
+        verify(tankRepo).deleteById(testId);
+    }
```

Comment:
^^this allows you to formulate test more precisely so you can test for a string as the return value